arlo-client/src/components/Audit/Setup/Participants/states.ts

```diff
@@ -1,53 +1,68 @@
-export default [
-  '',
-  'Alabama',
-  'Alaska',
-  'Arizona',
-  'Arkansas',
-  'California',
-  'Colorado',
-  'Connecticut',
-  'Delaware',
-  'Florida',
-  'Georgia',
-  'Hawaii',
-  'Idaho',
-  'Illinois',
-  'Indiana',
-  'Iowa',
-  'Kansas',
-  'Kentucky',
-  'Louisiana',
-  'Maine',
-  'Maryland',
-  'Massachusetts',
-  'Michigan',
-  'Minnesota',
-  'Mississippi',
-  'Missouri',
-  'Montana',
-  'Nebraska',
-  'Nevada',
-  'New Hampshire',
-  'New Jersey',
-  'New Mexico',
-  'New York',
-  'North Carolina',
-  'North Dakota',
-  'Ohio',
-  'Oklahoma',
-  'Oregon',
-  'Pennsylvania',
-  'Rhode Island',
-  'South Carolina',
-  'South Dakota',
-  'Tennessee',
-  'Texas',
-  'Utah',
-  'Vermont',
-  'Virginia',
-  'Washington',
-  'West Virginia',
-  'Wisconsin',
-  'Wyoming',
-]
+const states: { [key: string]: string } = {
+  AL: 'Alabama',
+  AK: 'Alaska',
+  AS: 'American Samoa',
+  AZ: 'Arizona',
+  AR: 'Arkansas',
+  CA: 'California',
+  CO: 'Colorado',
+  CT: 'Connecticut',
+  DE: 'Delaware',
+  DC: 'District Of Columbia',
+  FM: 'Federated States Of Micronesia',
+  FL: 'Florida',
+  GA: 'Georgia',
+  GU: 'Guam',
+  HI: 'Hawaii',
+  ID: 'Idaho',
+  IL: 'Illinois',
+  IN: 'Indiana',
+  IA: 'Iowa',
+  KS: 'Kansas',
+  KY: 'Kentucky',
+  LA: 'Louisiana',
+  ME: 'Maine',
+  MH: 'Marshall Islands',
+  MD: 'Maryland',
+  MA: 'Massachusetts',
+  MI: 'Michigan',
+  MN: 'Minnesota',
+  MS: 'Mississippi',
+  MO: 'Missouri',
+  MT: 'Montana',
+  NE: 'Nebraska',
+  NV: 'Nevada',
+  NH: 'New Hampshire',
+  NJ: 'New Jersey',
+  NM: 'New Mexico',
+  NY: 'New York',
+  NC: 'North Carolina',
+  ND: 'North Dakota',
+  MP: 'Northern Mariana Islands',
+  OH: 'Ohio',
+  OK: 'Oklahoma',
+  OR: 'Oregon',
+  PW: 'Palau',
+  PA: 'Pennsylvania',
+  PR: 'Puerto Rico',
+  RI: 'Rhode Island',
+  SC: 'South Carolina',
+  SD: 'South Dakota',
+  TN: 'Tennessee',
+  TX: 'Texas',
+  UT: 'Utah',
+  VT: 'Vermont',
+  VI: 'Virgin Islands',
+  VA: 'Virginia',
+  WA: 'Washington',
+  WV: 'West Virginia',
+  WI: 'Wisconsin',
+  WY: 'Wyoming',
+}
+
+export const labelValueStates = Object.keys(states).map(value => ({
+  value,
+  label: states[value],
+}))
+
+export default states
```

Comment:
Why not use an `enum`? It'd be something like this:

```ts
enum USState {
  AL = 'Alabama'
  …
}
```

You end up with pretty much the same object, but more type-safe.

Also, we seem to have used different values for the states: [`USState` in models.py](https://github.com/votingworks/arlo/blob/f75ce2a5e388ec95bde93a3a2ec75f1626bc6990/arlo_server/models.py#L370-L428). Perhaps we should stick this in a text file and generate backend and frontend models? Or the frontend could just ask the backend for the list. This seems to be a pretty complete list: https://en.wikipedia.org/wiki/List_of_U.S._state_abbreviations. I'd probably include everything but the ones listed "obsolete". @mcchilders, does it matter what we include or exclude here?

Reply:
It needs all 50 states, none of the territories for now.

Reply:
@eventualbuddha I just tried an enum, but it broke the generation of the array of labels and values needed for the dropdown. It doesn't look like enums play nicely with mapping, possibly? Do you have an example of how this would work?

On the other hand, we are never going to be actually dealing with these values directly. They're always going to be behind the scenes, and interacted with as plain strings. So I'm not sure they need to be more type-safe than they already are. Are you thinking of a particular situation where it would be needed?